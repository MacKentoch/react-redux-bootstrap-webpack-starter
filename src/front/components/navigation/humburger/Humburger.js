// @flow

// #region imports
import React from 'react';
// #endregion

const Humburger = () => {
  return (
    <button
      className="navbar-toggle collapsed"
      type="button"
      data-toggle="collapse"
      data-target="#bs-example-navbar-collapse-1"
    >
      <span className="sr-only">Toggle navigation</span>
      <span className="icon-bar" />
      <span className="icon-bar" />
      <span className="icon-bar" />
    </button>
  );
};

// #region statics
Humburger.displayName = 'Humburger';
// #endregion

export default Humburger;
