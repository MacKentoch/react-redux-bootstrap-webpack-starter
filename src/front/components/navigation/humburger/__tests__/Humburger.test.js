// @flow

// #region imports
import React from 'react';
import renderer from 'react-test-renderer'; // needed both for snpashot testing but also to prevent errors from enzyme
import Humburger from '../Humburger';
// #endregion

describe('Humburger component', () => {
  it('renders as expected', () => {
    const component = renderer
      .create(
        <div>
          <Humburger />
        </div>,
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
