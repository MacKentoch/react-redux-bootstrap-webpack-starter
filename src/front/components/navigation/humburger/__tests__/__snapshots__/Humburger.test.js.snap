// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Humburger component renders as expected 1`] = `
<div>
  <button
    className="navbar-toggle collapsed"
    data-target="#bs-example-navbar-collapse-1"
    data-toggle="collapse"
    type="button"
  >
    <span
      className="sr-only"
    >
      Toggle navigation
    </span>
    <span
      className="icon-bar"
    />
    <span
      className="icon-bar"
    />
    <span
      className="icon-bar"
    />
  </button>
</div>
`;
